config.xml

```diff
@@ -48,6 +48,7 @@
         <preference name="UseSwiftLanguageVersion" value="4.2" />
         <allow-intent href="itms:*" />
         <allow-intent href="itms-apps:*" />
+        <preference name="DisallowOverscroll" value="true" />
```

Comment:
What is the motivation behind this? In general, I'm against this because it needs
- to behave consistently with other iOS apps (to don't break iOS users experience)
- to behave consistently with the web version of wallet

The next time would be better to write "why something is done" instead of "what is done" in the commit message. For example, "fix(iOS): disable overscroll to fix XYZ" or just "fix(iOS): XYZ" 🤔

Reply:
> What is the motivation behind this? In general, I'm against this because it needs
> 
>     * to behave consistently with other iOS apps (to don't break iOS users experience)
> 
>     * to behave consistently with the web version of wallet
> 

Totally agreed on not breaking the UX of the native platform.

Disallowing overscroll was in attempt to achieve the scrolling behaviour of the interactive prototype. In combination with the proposal to use `sticky` instead of `fixed`. 

This is how it behaves without overscroll disabled:

https://user-images.githubusercontent.com/482351/112992432-d6244d00-9170-11eb-83eb-783706952edd.mp4

If we want to achieve the native UX + consistency with the web wallet + resemblance with the interactive prototype - then I guess the element should be positioned `fixed`, so only the area beneath is scrolled/overscrolled. 

Any better solutions?

> 
> The next time would be better to write "why something is done" instead of "what is done" in the commit message. For example, "fix(iOS): disable overscroll to fix XYZ" or just "fix(iOS): XYZ" 🤔

It is consistent with the repo commit message history, in my opinion.

Indeed an explanation maybe in the PR description or commit body would've been helpful and it is my bad for not including it.

Reply:
I synced with @imollov  and I agree with his suggestion to keep the native functionality but make fixed both the header and the following component. The behavior visible in the video just looks strange. 
Also I noticed a few other visual bugs which are described in this screenshot:
![Screenshot_1](https://user-images.githubusercontent.com/12225410/113416627-67dac700-93ca-11eb-9b16-5f2dadd43bbc.png)

Reply:
Is this still actual in regards to the latest changes in the develop branch?

Reply:
The side remarks by @onvisions are fixed in develop and #951. 

What remains unsolved is the scrolling behavior on iOS. Simply put - the plate should not scroll by design and it currently does.

Although with the latest changes to support web layout I don't see an easy way to achieve it.